runelite-client/src/main/java/net/runelite/client/plugins/motherlode/MotherlodeSession.java

```diff
@@ -108,25 +114,64 @@ void incrementGemFound(int gemID)
 
 	void updateOreFound(int item, int count)
 	{
+		Instant now = Instant.now();
+		Duration timeSinceStart;
+
 		switch (item)
 		{
 			case ItemID.GOLDEN_NUGGET:
 				nuggetsFound += count;
+
+				timeSinceStart = Duration.between(recentPayDirtMined, now);
```

Comment:
Can't `recentPayDirtMined` be null here?

Reply:
I believe it can potentially be null, but it would also be better for the time since start to be at the time of mining for better numbers. However its probably necessary to make a check in each case to prevent the null case